### DISCUSSION TITLE
Project Translation

### DISCUSSION THREAD

**Discussion by shuvashish76** [KRTirtho/spotube#66] [Ideas]
I've seen OSS projects use these sites for translation purpose
- [Crowdin](https://crowdin.com/)
- [Transifex](https://www.transifex.com/)
- [Weblate](https://weblate.org/en/)

I think Weblate is better as it's [opensource](https://github.com/WeblateOrg/weblate).

Features: https://weblate.org/en/features/
![Screenshot_2021-12-05-21-15-09](https://user-images.githubusercontent.com/48226312/144753433-4fc61e84-f12d-42d0-885e-1b9abd399061.jpeg)
https://weblate.org/en/hosting/

**Comment by comradekingu** [comment]
Another +1 for Weblate, and a solid -1 for the other two.
Happy to help if you drop the CoC :)

**Comment by KRTirtho** [comment]
Its gonna take a bit time but we'll implement it together. I'm kind of new to weblate so drop every knowledge bomb you have about it to help the project

**Reply by comradekingu** [reply]
@KRTirtho 
https://docs.weblate.org/en/latest/formats.html#translation-types-capabilities are the various supported file formats.
Setup isn't the most polished part, but once up and running it is all good.
I can help with that if the CoC is removed.
https://weblate.bubu1.eu/ and https://hosted.weblate.org/hosting/ are available.
My username on those is "kingu".
It can also be self-hosted.

The similar projects are already on Weblate
https://hosted.weblate.org/projects/skytube/
https://hosted.weblate.org/projects/piped/
https://hosted.weblate.org/projects/piped-material/
https://hosted.weblate.org/projects/newpipe/
https://hosted.weblate.org/projects/pstube/
https://hosted.weblate.org/projects/peertube/
https://hosted.weblate.org/projects/libretube/
https://hosted.weblate.org/projects/free-tube/

What are the different clients, and do they share stringbase files?
